addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -107,6 +107,7 @@ var FieldMany2One = AbstractField.extend({
         'click': '_onClick',
     }),
     AUTOCOMPLETE_DELAY: 200,
+    SEARCH_MORE_LIMIT: 480,
```

Comment:
maybe use an `ir.parameter`

`SEARCH_MORE_LIMIT: int(env[`ir.parameter`].get_paramter('SEARCH_MORE_LIMIT')) or 480`

Reply:
+1 would be a good moment to have it configurable.

Reply:
If it should be a parameter, then I would put it at the level of the arch of the view, not global, so that if could be change on view by view basis, and also be extensible.